wallet/src/de/schildbach/wallet/ui/RestoreWalletFromSeedActivity.kt

```diff
@@ -160,12 +160,4 @@ class RestoreWalletFromSeedActivity : RestoreFromFileActivity() {
             show()
         }
     }
-
-    override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
-        super.onActivityResult(requestCode, resultCode, data)
-        if (requestCode == SET_PIN_REQUEST_CODE && resultCode == Activity.RESULT_OK) {
-            setResult(Activity.RESULT_OK)
-            finish()
-        }
-    }
```

Comment:
Using `finishAffinity` will remove the need to setResult to tell previous Activities to close.